xgi/convert.py

```diff
@@ -15,6 +17,8 @@
     "from_bipartite_pandas_dataframe",
     "from_incidence_matrix",
     "to_incidence_matrix",
+    "from_bipartite_network",
+    "to_bipartite_network"
```

Comment:
I'm usually in favor of shorter names. Is there any reason not to call this `from_bipartite`? Or even `from_bipartite_graph` which is still shorter.

Reply:
I would leave the `_graph` at least, just to avoid the unexpected --but you never know-- "What's a bipartite hypergraph?"

Reply:
Okay I changed `_network` to `_graph`.